--- doc/html-stylesheet.xsl.orig
+++ doc/html-stylesheet.xsl
@@ -1,6 +1,6 @@
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version='1.0'>
 
-  <xsl:import href="/usr/share/xml/docbook/stylesheet/nwalsh/html/docbook.xsl"/>
+  <xsl:import href="/usr/share/xsl/docbook/xhtml/docbook.xsl"/>
 
   <xsl:param name="toc.section.depth">3</xsl:param>
   <xsl:param name="annotate.toc">0</xsl:param>
